### DISCUSSION TITLE
Duplicate Files still showing after rescan

### DISCUSSION THREAD

**Discussion by utubeuser234** [stashapp/stash#5175] [Q&A] [answered]
I have folders that were initially scanned in one folder now moved, and re-scanned, but now it shows 2 files. One with the original file location and one with the new file location. I re-scanned the old folder after moving the files but it still shows both file paths. Right now I have to make the one that works the primary and delete the other file. Is there an easier way to remove the duplicates? These are gallery folders that have 100's of files.

**Comment by feederbox826** [comment, marked as answer]
Clean will remove files that have been removed, run a scan and a dry clean to make sure that you don't accidentally remove anything you wanted to keep
